### DISCUSSION TITLE
Custom Site Icon?

### DISCUSSION THREAD

**Discussion by 0w1Knight** [Sanjeet990/Astroluma#8] [Ideas]
Hey there - Is it possible to customize the site icon? I have a logo I use for my server I'd love to use for the home page.

**Comment by Sanjeet990** [comment]
You mean this one?

![image](https://github.com/user-attachments/assets/a0c9f8ea-1077-4ec8-a440-d02be27c4d87)

**Reply by Node815** [reply]
I believe this to be the case or the favicon or both.

**Comment by Sanjeet990** [comment]
Functionality added in Astroluma v1.0.2
